pipelines/imputation/scientificValidation/README.md

```diff
@@ -59,3 +59,42 @@ we usually use `gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_asse
 #### Outputs
 * output_vcf - reheadered vcf file
 * output_vcf_index
+
+## BeagleValidation
+### Docker
+Be sure to update the jars in `beagle_jars` if you
+want to create a new image with updated jars
+
+When building the docker image and you are on a M1 mac machine
+, be sure to add `--platform=linux/amd64` to your docker build command
+
+### Purpose
+This wdl is meant to help scientifically validate
+an imputation output.  This outputs a tsv file that
+contains data that can be used to see how well imputation
+performed like r^2 and binned by AC.
+
+#### Inputs
+* ref_panel_vcf
+* ref_panel_vcf_index
+* truth_vcf - vcf containing wgs data for sample(s)
+that was imputed 
+* truth_vcf_index
+* test_vcf - output from imputation workflow
+* test_vcf_index
+
+#### Outputs
+* imputed_r2_output - tsv with the following description
+```
+Six fields are printed for each frequency bin:
+
+ 1) MIN_AC:  the minimum reference allele count for this frequency bin
```

Comment:
is this AC calculated from the reference panel or the test data?

Reply:
i went back and forth with megan on this a few days ago haha.  it _has_ to be reference based on the numbers where i ran it on in the example workflow.  Also chatted with chris about it [here](https://broadinstitute.slack.com/archives/C06Q4BSMS14/p1728332236538889).  This comment comes directly from the jars usage though (theres no way i could come up with this)